WORKSPACE.yaml

```diff
@@ -7,7 +7,7 @@ defaultArgs:
   jbMarketplacePublishTrigger: "false"
   publishToJBMarketplace: true
   localAppVersion: unknown
-  codeCommit: 64ebb59829c0d3998ebf6b69308aafd49362d29c
+  codeCommit: e9a328220ac3a6f7216451151438b145437b1e44
```

Comment:
this one should go to gp-code/main before merging?

Reply:
yep

Reply:
Actually nope, we need blobserve to be deployed first before we can update gp-code/main